tasks/main.yml

```diff
@@ -50,7 +48,7 @@
 
 # Logging into DNAnexus account
 - name: Log in to DNAnexus account if dx_user_token is provided
-  shell: source /opt/dx-toolkit/environment && dx login --token {{ item.dx_user_token if 'dx_user_token' in item else dx_token }} --noprojects
+  shell: dx login --token {{ item.dx_user_token if 'dx_user_token' in item else dx_token }} --noprojects
```

Comment:
why dont we source environment anymore?

Reply:
it is not required to load environment anymore when dx-toolkit is installed via pip3